apps/challenges/views.py

```diff
@@ -42,6 +42,12 @@ def challenge_list(request, challenge_host_team_pk):
         return paginator.get_paginated_response(response_data)
 
     elif request.method == 'POST':
+
+        if not ChallengeHost.objects.filter(user=request.user).filter(team_name=challenge_host_team_pk):
```

Comment:
we do create challenge host object when ever any new challenge host team is created. you can have a look at api `create_challenge_host_team` in `hosts/views.py`